### PR TITLE
Fix Absolute Control tooltip.

### PR DESCRIPTION
**Pull request by McGiverGim**
Fixes the tooltip for Absolute Control.

### CODE REVIEW

File: locales/en/messages.json [hunk 1]
```diff
@@ -3117,7 +3117,7 @@
         "message": "Absolute Control"
     },
     "pidTuningAbsoluteControlGainHelp": {
-        "message": "This feature solves some underlying problems of I Term Rotation and should hopefully replace it at some point. The feature accumulates the absolute gyro error in quad coordinates and mixes a proportional correction into the setpoint. It needs to have the iterm_relax enable for RPY."
+        "message": "This feature solves some underlying problems of $t(pidTuningItermRotation.message) and should hopefully replace it at some point. This feature accumulates the absolute gyro error in quad coordinates and mixes a proportional correction into the setpoint. It needs to have $t(pidTuningItermRelax.message) enabled for $t(pidTuningItermRelaxAxesOptionRP.message). If you combine this feature with $t(pidTuningIntegratedYaw.message), you can set $t(pidTuningItermRelax.message) enabled for $t(pidTuningItermRelaxAxesOptionRPY.message)."
```
Comment (mikeller) [inline locales/en/messages.json]: Shouldn't the end of this be "you **should** set $t(pidTuningItermRelax.message) enabled for $t(pidTuningItermRelaxAxesOptionRPY.message)."?

Comment (McGiverGim) [reply locales/en/messages.json]: I think that not, is entirely optional to the user use RP or RPY. Or at least this is what I understood to Joe in Slack. @joelucid can you confirm if this text is right?

Comment (mikeller) [reply locales/en/messages.json]: Please disregard my earlier comment - I had read the placeholders in the wrong order.

**Comment by McGiverGim** [comment]
Sorry @mikeller it seems that it needs AirMode ON too. so added this info to the text...
image

File: locales/en/messages.json (hunk 1 above)
Comment (joelucid) [reply locales/en/messages.json]: Yeah the message is correct.

**Pull request merged**